src/main/java/com/google/flourbot/entity/action/sheet/SheetReadColumnAction.java

```diff
@@ -0,0 +1,29 @@
+package com.google.flourbot.entity.action.sheet;
+
+import com.google.flourbot.api.DriveCloudSheet;
+import com.google.flourbot.entity.action.ActionType;
+
+public final class SheetReadColumnAction extends SheetAction {
+
+  private final ActionType actionType = ActionType.SHEET_READ_COLUMN;
+  private final String column;
+
+  public SheetReadColumnAction(String sheetUrl, String column) {
+    super(sheetUrl);
+    this.column = column;
+  }
+
+  public SheetReadColumnAction(String sheetUrl, int column) {
+    // Overload
+    super(sheetUrl);
+    this.column = DriveCloudSheet.toAlphabetic(column + 1);
```

Comment:
not clear why we have +1 here.

Reply:
The function is A-> 0 but here we want A->1. Not ideal